src/main/java/com/gmail/goosius/siegewar/utils/SiegeWarNotificationUtil.java

```diff
@@ -57,16 +59,22 @@ public static void informSiegeParticipants(Siege siege, String message) {
 
 			//Inform required towns and nations
 			for(Nation nationToInform: nationsToInform) {
-				TownyMessaging.sendPrefixedNationMessage(nationToInform, message);
+				for (Translatable line : message) 
+					if (line != null)
+						TownyMessaging.sendPrefixedNationMessage(nationToInform, line);
```

Comment:
- As per comment in discord, just wondering if it is ok from a server performance point of view to now send 2 separate messages (in some cases), instead of just one?